### DISCUSSION TITLE
Start in normal vi mode

### DISCUSSION THREAD

**Discussion by dimas-cyriaco** [nushell/nushell#6947] [Q&A] [answered]
Is there a way to configure nushell to start in vi normal mode?

**Comment by sholderbach** [comment, marked as answer]
Not yet, small changes are necessary in https://github.com/nushell/reedline and the config parsing/ repl setup necessary to make this possible.
But no bigger hurdle to get this.

**Reply by dimas-cyriaco** [reply]
Ok, thanks.

**Reply by weilbith** [reply]
@sholderbach any update on this? 🙂

**Reply by sholderbach** [reply]
Sadly, as far as I can tell we have not made progress on that part of the vi mode.

**Reply by weilbith** [reply]
In Zsh I do something like this:

```zsh
zle-line-init() { zle -K vicmd; }
zle -N zle-line-init
```

Is something that primitive possible with nushell? 🤔
